content/articles/where-incorporate-eu-citizen.md

```diff
@@ -0,0 +1,163 @@
+Title: Where to incorporate if you are a EU citizen
+Date: 2016-07-10
+Short_summary: We have lots of options as to where incorporate as EU citizens, let's look at them.
+Category: Business
+Authors: Vincent
+
+*Reading time: ~10 minutes.*
+
+We are very lucky as EU citizens to have access to 28 (well, soon 27 apparently) countries.
+To give a bit of context to this blog post, We Are Wizards is currently a company based in the UK, founded by 2 French and one German, all living in London at the moment.
+
+This article will try to be generalistic but we have a few requirements of our own that influenced this research. There might be inaccuracies in this article so don't take my word for it and contact companies/government in the countries before pulling the trigger if you decide to do so.
```

Comment:
"While this post tries to be general it is guided by our own requirements"

"before making a decision"